src/de/willuhn/jameica/fibu/io/report/VelocityReportBuchungList.java

```diff
@@ -55,14 +55,12 @@ protected VelocityReportData getData(ReportData data) throws Exception
     Konto endKonto   = data.getEndKonto();
     
     list.setOrder("order by datum");
+    //Split-Hauptbuchungen rausfiltern
+    list.addFilter("NOT EXISTS(SELECT 1 FROM buchung b WHERE b.split_id = buchung.id)");
```

Comment:
Sowohl hier als auch in VelocityReportKontoAuszug kommt "list" aus "jahr.getHauptBuchungen(....)". Könnte man das zusätzliche Filterkriterium "NOT EXISTS(SELECT 1 FROM buchung b WHERE b.split_id = buchung.id)" nicht direkt dort mit unterbringen? Oder ggf. "jahr.getHauptBuchungen(...)" um einen Boolean-Parameter "noSplitHauptbuchungen" erweitern? Dann hätte man das an zentraler Stelle und kann sicher sein, dass bei allen Aufrufern korrekt berücksichtigt ist, ob die Split-Hauptbuchungen enthalten sein sollen oder nicht.